protos/mission/mission.proto

```diff
@@ -92,6 +91,11 @@ message MissionItem {
     }
 }
 
+message MissionProgress {
```

Comment:
> Responses cannot have more than 1 return parameter

This is relating, right?

Reply:
Yes. An observable returns a stream of objects of the same type or an error.

If you look at the swift implementation, actually, the two return parameters of this response were actually packaged inside a tuple anyway.

Reply:
Okay, I that's what I was thinking!